moniker_src/data_moniker_service.cpp

```diff
@@ -81,218 +96,202 @@ namespace ni
             {
                 ptr = (*it).second;
             }
-            writers.push_back(EndpointInstance(ptr, reinterpret_cast<void*>(instance)));
+            writers->push_back(EndpointInstance(ptr, reinterpret_cast<void*>(instance)));
         }
     }
 
     //---------------------------------------------------------------------
     //---------------------------------------------------------------------
-    void MonikerServiceImpl::RunSidebandReadWriteLoop(string sidebandIdentifier, ::SidebandStrategy strategy, EndpointList& readers, EndpointList& writers, bool initialClientWrite)
+    void DataMonikerService::RunSidebandReadWriteLoop(string sidebandIdentifier, ::SidebandStrategy strategy, EndpointList* readers, EndpointList* writers)
     {
+        std::cout << "Enter RunSidebandReadWriteLoop" << std::endl;
+
     #ifndef _WIN32
         if (strategy == ::SidebandStrategy::RDMA_LOW_LATENCY ||
             strategy == ::SidebandStrategy::SOCKETS_LOW_LATENCY)
         {
+            pid_t threadId = syscall(SYS_gettid);
+            ::SysFsWrite("/dev/cgroup/cpuset/LabVIEW_tl_set/tasks", std::to_string(threadId));
```

Comment:
What is this doing?  What about systems that are not using LabVIEW.  I don't think we should tie the grpc sideband repo to LabVIEW.

Reply:
I'm not sure why this was added. I have removed this change as I agree we should not tie sideband to anything related to LABVIEW.